manual/manual.adoc

```diff
@@ -107,6 +112,12 @@ and killed monsters come back to life after around 40 seconds.
 [[savegame]]
 === Loading and saving the game
 
+[**Warning:** the original Doom program has a bug that makes it crash
```

Comment:
I think this would fit better at the end of the section, since it's only applicable to a single source port.

Reply:
I always assumed there were other ports out there that did this as well. If not then this is a good deal easier to write.